admin/admin-strings.php

```diff
@@ -79,22 +78,39 @@ public static function &get_strings() {
 			}
 
 			foreach ( $widgets as $widget ) {
-				// Nothing can be done if the widget is created using pre WP2.8 API :(
-				// There is no object, so we can't access it to get the widget options
+				// Nothing can be done if the widget is created using pre WP2.8 API. There is no object, so we can't access it to get the widget options.
 				if ( ! isset( $wp_registered_widgets[ $widget ]['callback'][0] ) || ! is_object( $wp_registered_widgets[ $widget ]['callback'][0] ) || ! method_exists( $wp_registered_widgets[ $widget ]['callback'][0], 'get_settings' ) ) {
 					continue;
 				}
 
 				$widget_settings = $wp_registered_widgets[ $widget ]['callback'][0]->get_settings();
 				$number = $wp_registered_widgets[ $widget ]['params'][0]['number'];
 
-				// Don't enable widget translation if the widget is visible in only one language or if there is no title
-				if ( empty( $widget_settings[ $number ]['pll_lang'] ) ) {
-					if ( isset( $widget_settings[ $number ]['title'] ) && $title = $widget_settings[ $number ]['title'] ) {
+				// Don't enable widget translation if the widget is visible in only one language or if there is no title.
+				if ( ! empty( $widget_settings[ $number ]['pll_lang'] ) ) {
+					continue;
+				}
+
+				// Widget title.
+				if ( isset( $widget_settings[ $number ]['title'] ) ) {
+					$title = $widget_settings[ $number ]['title'];
+					if ( ! empty( $title ) ) {
 						self::register_string( self::$default_strings['widget_title'], $title, 'Widget' );
 					}
+				}
+
+				// Text of the Widget text.
+				if ( isset( $widget_settings[ $number ]['text'] ) ) {
+					$text = $widget_settings[ $number ]['text'];
+					if ( ! empty( $text ) ) {
+						self::register_string( self::$default_strings['widget_text'], $text, 'Widget', true );
+					}
+				}
```

Comment:
```suggestion
				// Text of the Widget text.
				if ( ! empty( $widget_settings[ $number ]['text'] ) ) {
					self::register_string( self::$default_strings['widget_text'], $widget_settings[ $number ]['text'], 'Widget', true );
				}
```
Follows previous @szepeviktor suggestion.